common/corerequestcontext.go

```diff
@@ -95,9 +95,15 @@ func CoreRequestContextMiddleware(logger *logrus.Logger) func(next http.Handler)
 			ctx = internal.AddResponseBodyMonitorToContext(ctx)
 			defer internal.CheckForUnclosedResponses(ctx, entry)
 
+			reqLogger, entry := internal.NewRequestLogger(entry, r)
+			w = reqLogger.ResponseWriter(w)
+			defer reqLogger.FlushLog()
+			LoggerToContext(ctx, logger, entry)
```

Comment:
the return value of this function is the context with the logger, you need to be using this value

Reply:
addressed